### DISCUSSION TITLE
Auto pages/layouts/modules Imports Like Nuxt

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [quasarframework/quasar#16007] [Ideas / Proposals] [answered]
Any plan to implement the auto imports of pages, layouts, and modules like in Nuxt?
I've searched and didn't find anything, so I always have to implement it by myself using `vite-plugin-vue-layouts`, `unplugin-auto-imports`, `unplugin-vue-components`, and `unplugin-vue-router` when working with Quasar + Vite CLI. Thanks.

**Comment by angelhdzmultimedia** [comment]
💔😔

**Comment by rstoenescu** [comment, marked as answer]
Yes. There are plans. It will come.
